test/storage.js

```diff
@@ -0,0 +1,21 @@
+/* global module */
```

Comment:
Thought about that myself. Almost thinking I might move this and the tests to a `test/node` directory. This helper could then be in `test/node/helpers` (since it's not really a fixture).